websockets/constants.js

```diff
@@ -6,6 +6,9 @@ var PORT_SSL = "{{ports[wss][0]}}";
 var SCHEME_DOMAIN_PORT;
 if (location.search == '?wss') {
   SCHEME_DOMAIN_PORT = 'wss://{{host}}:' + PORT_SSL;
+} else if (location.search == '?http2') {
+  var PORT_H2 = "{{ports[h2][0]}}";
```

Comment:
Shall we define this at the top of the file for consistency?

Reply:
done